app/javascript/packs/application.js

```diff
@@ -0,0 +1,18 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
+require("jquery3");
```

Comment:
```suggestion
require("jquery");
```
jquery thôi mới chạy dc ô ạ